submission/persistence/src/main/kotlin/ac/uk/ebi/biostd/persistence/util/SubmissionFilterSpecification.kt

```diff
@@ -0,0 +1,57 @@
+package ac.uk.ebi.biostd.persistence.util
+
+import ac.uk.ebi.biostd.persistence.model.Submission
+import ac.uk.ebi.biostd.persistence.model.User
+import org.springframework.data.jpa.domain.Specification
+
+class SubmissionFilterSpecification(userId: Long, filter: SubmissionFilter) {
+    var specification: Specification<Submission>
+
+    init {
+        specification = Specification.where(
+            withUser(userId)).and(withVersionGreaterThan(0))
```

Comment:
this one fit in a single line according to  coding convention  (check docs folder for more reference)

```
specification = Specification.where(withUser(userId)).and(withVersionGreaterThan(0))
```